src/components/todosList.tsx

```diff
@@ -0,0 +1,22 @@
+import { useContext } from 'react';
+import { Todo } from '../types/Todo';
+import { TodosItem } from './todosItem';
+import { TodosContext } from './todosContext';
+
+interface ListProps {
+  items: Todo[];
+}
+
+export const TodosList: React.FC<ListProps> = ({ items }) => {
+  const { todos } = useContext(TodosContext);
+
+  return (
+    !!todos.length && (
+      <>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings